tests/test_compdat.py

```diff
@@ -181,6 +182,53 @@ def test_tstep():
     assert "2001-05-07" in dates
 
 
+@pytest.mark.parametrize("test_input,expected", WELOPEN_CASES)
```

Comment:
I am not sure, but I would have put this entire function into `test_welopen.py`. It should be ok to split the testing of the compdat-module into more than one file (even though this is the first file doing that).

Reply:
Move function. Kept "test_welopen" in here, as that tests directly a function in the compdat.py file.... not entirely sure though. What do you think?

Reply:
fine!